docs/tutorials/avro.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p>when I tried it out, the output consumed a lot of space. Can we limit the number of records displayed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/io/pull/1250/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
thanks, updated.